mllib/src/main/scala/org/apache/spark/ml/ann/Layer.scala

```diff
@@ -374,6 +380,22 @@ private[ann] trait TopologyModel extends Serializable {
   def predict(data: Vector): Vector
 
   /**
+   * Raw prediction of the model
+   *
+   * @param data input data
+   * @return raw prediction
+   */
+  def predictRaw(data: Vector): Vector
+
+  /**
+   * Probability of the model
+   *
+   * @param data input data
```

Comment:
"input data" sounds like input feature values, which is not correct.  Why not match the ProbabilisticClassifier API?